python/grass/pygrass/raster/__init__.py

```diff
@@ -13,21 +13,18 @@
 
 libgis.G_gisinit("")
 
-#
-# import pygrass modules
-#
-from grass.pygrass.errors import must_be_open
-from grass.pygrass.gis.region import Region
-from grass.pygrass import utils
-
-#
-# import raster classes
-#
-from grass.pygrass.raster.abstract import RasterAbstractBase
-from grass.pygrass.raster.raster_type import TYPE as RTYPE, RTYPE_STR
-from grass.pygrass.raster.buffer import Buffer
-from grass.pygrass.raster.segment import Segment
-from grass.pygrass.raster.rowio import RowIO
+if True:
```

Comment:
Leftover "if True...import".

Reply:
The workflow definitively feels like there is a lot of steps. Try GitHub CLI (gh) to reduce some of them.